packages/client/src/runtime/core/compositeProxy/forwardTo.ts

```diff
@@ -0,0 +1,20 @@
+import type { CompositeProxyLayer } from './createCompositeProxy'
+
+/**
+ * Composite proxy layer that forwards all reads
+ * to provided object
+ *
+ * @param object
+ * @returns
+ */
+export function forwardTo(object: object): CompositeProxyLayer {
```

Comment:
I would prefer a different name like objectToCompositeProxyLayer. Much longer put more descriptive imo.

Reply:
Renamed to `addObjectProperties`: more descriptive than `forwardTo`, but also consistent with `addProperty` layer. Let me know if you still prefer different name.

Reply:
That's better, thanks.